server/test/utils/setup.js

```diff
@@ -0,0 +1,59 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+const path = require('path');
+const chaiHttp = require('chai-http');
+
+const {validCredentials} = require('../mocks/user')
+
+chai.should();
+chai.use(chaiHttp);
+
+const dotEnvPath = path.resolve(__dirname + './../../.env');
+require('dotenv').config({ path: dotEnvPath });
+
+const app = require('../../app.js');
+
+exports.AssertPostRequest = (path , payload, expectedStatusCode, done) => {
+    chai.request(app)
+    .post(path)
+    .send(payload)
+    .end((err, res) => {
+        if(err)
+            throw err;
+        res.should.have.status(expectedStatusCode);
+        done();
+    });
```

Comment:
I don't know if we really need these helpers, because we'd probably want to check side effects (such as did we actually add to the database), and etc. - in the API test calls, and not just the response status

Reply:
I think using these helpers allows us to reuse the code in an efficient way (status code validation will be done for all endpoint tests) and create clean test files.
To test the side effects I thought to add a function as an optional parameter to helpers containing extra validations and to execute it after validation on the status code.

how do you think?